src/libLookup/Lookup.cpp

```diff
@@ -2981,6 +2992,8 @@ bool Lookup::ProcessSetTxBlockFromSeed(
         WARNING,
         "The lowBlockNum is higher than highblocknum, maybe DS epoch ongoing");
     cv_setTxBlockFromSeed.notify_all();
+    // cv_setRejoinRecovery.notify_all(); // No need to notify as we need to
```

Comment:
what constant denotes 180 seconds? maybe mention that

Reply:
Removed the line as it was initially kept for my understanding